### DISCUSSION TITLE
THANK YOU

### DISCUSSION THREAD

**Discussion by 4o4R** [iBicha/playlet#510] [General]
THANK YOU! You are awesome for this. All of you contributing. That is all.

**Comment by JCollins2048** [comment]
Gonna necro this just to say: you work too hard for too little pay.

Thank you for all your hard work, iBicha.  *Playlet* continues to be a godsend for those of us who hate ads.
(And tracking, I suppose.  Heh.)

**Reply by iBicha** [reply]
Thank you for the kind words!
> you work too hard for too little pay

I mean, I don't make anything out of Playlet, this app is not monetized in anyway. In fact it's costing me money to run a server to decipher video streams. But that's fine, my day job allows me to do that.
